Hypershift 0.6 changes (#2420)
timescale/docs: use-timescale/migration/about-hypershift.md

To see all available configuration options for hypershift, use this command:

```bash
docker run -ti timescale/hypershift:0.6 --help
```

### The hypershift clone tool

> You can use the hypershift `clone` command to preserve a copy of your database

Reviewer: ```suggestion
You can use the hypershift `clone` command to make a copy of your database
```
Does `make` works? I think it simplifies the sentence.

Author: "make" implies creating something new, which I'm not sure is actually what is happening ...

Author: "create" again, means something new.

Reviewer: Oh, then lets go with preserve. The reason why I suggested create is Hypershift is responsible to create a copy of the database to the target.

> in the source location. With this command, instead of moving the contents off
> the source and into the target, hypershift makes a full copy of the contents of
> your source database, and moves it into the source database. You can add
> parameters to the `clone` command to filter which contents to copy.
> 

Reviewer: Can we simplify to this?

```suggestion
in the source location. Hypershift makes a full copy of the contents of
your source database, and moves it into the target database. You can use
parameters in the `clone` command to filter which contents to copy.
```

|Short command|Long command|Example use|Description|
|-|-|-|-|
|-s|--source|-s <SOURCE_URI>|URI to the source database|
|-t|--target|-t <TARGET_URI>|URI to the target database|
||--source-password|--source-password|Password for the source database|
||--target-password|--target-password|Password for the target database|

> ||--parallel|--parallel <NUMBER\>|Number of workers that can simultaneously fetch data from the target database|

Reviewer: Is the `\` in `<NUMBER\>` intentional? There is also another one at 92, `<PATH\>`
